plenum/server/monitor.py

```diff
@@ -340,6 +371,13 @@ def calculateTraffic(self):
         currNetwork /= 1024
         return currNetwork
 
+    @staticmethod
+    def create_throughput_measurement(config, start_ts=time.perf_counter()):
```

Comment:
Why this method is in EMAThroughputMeasurement? It should be either in ThroughputMeasurement or in a separate factory/factory function.

Reply:
This method is contained in `Monitor` class.